dataland-frontend/tests/e2e/fixtures/frameworks/sme/SmePreparedFixtures.ts

```diff
@@ -10,67 +14,44 @@ import { generateNaceCodes } from "@e2e/fixtures/common/NaceCodeFixtures";
  */
 export function generateSmePreparedFixtures(): Array<FixtureData<SmeData>> {
   const preparedFixtures = [];
-  preparedFixtures.push(manipulateFixtureForYearWithMultipleSectors(generateSmeFixtures(1)[0], "2023"));
-  preparedFixtures.push(manipulateFixtureForMaximumAddress(generateSmeFixtures(1)[0]));
-  preparedFixtures.push(manipulateFixtureForMinimumAddress(generateSmeFixtures(1)[0]));
+  preparedFixtures.push(manipulateFixtureForSpecificTest(generateSmeFixtures(1)[0]));
+  preparedFixtures.push(manipulateFixtureForNoNullFields(generateSmeFixtures(1, 0)[0]));
   return preparedFixtures;
 }
 
 /**
- * Sets the company name, reporting period and some data in the fixture data to specific values needed for tests.
+ * Sets the company name, reporting period and some data in the fixture data to specific values needed for a specific
+ * test.
  * @param input Fixture data to be manipulated
- * @param year the year as a number
  * @returns the manipulated fixture data
  */
-function manipulateFixtureForYearWithMultipleSectors(input: FixtureData<SmeData>, year: string): FixtureData<SmeData> {
-  input.companyInformation.companyName = "SME-year-" + year;
-  input.reportingPeriod = year;
-  input.t.general.basicInformation.sector = generateNaceCodes(2);
+function manipulateFixtureForSpecificTest(input: FixtureData<SmeData>): FixtureData<SmeData> {
```

Comment:
unspecific function name. But I don't have a better suggestion. I'll just close this soonish if you don't have an enlightenment regarding that name.

Reply:
Tatsächlich hinterfrage ich, ob man dieses prepared fixture überhaupt noch braucht.
Denn der sme panel test ist ja rausgeflogen.
Deswegen entferne ich dieses prepared fixture jetzt mal komplett und pushe.

In dem einen Test, wo es aktuell benutzt wird, scheint es egal zu sein, was in dem dataset steht. Daher schreibe ich diesen Test so um, dass er einfach ein "normales" sme fixture benutzt, und kein prepared fixture.

Wenn trotzdem alles grün ist, kannst du diese conversation schließen.

Reply:
👍